test/unit/reporter.spec.js

```diff
@@ -23,7 +23,7 @@ describe('reporter', () => {
     beforeEach(() => {
       emitter = new EventEmitter()
       formatError = m.createErrorFormatter({ basePath: '', hostname: 'localhost', port: 8080 }, emitter)
-      sandbox = sinon.sandbox.create()
+      sandbox = sinon.createSandbox()
```

Comment:
AFAIK sinon is now sandbox by default, so it should be ok to remove this line and use sinon directly